integration_tests/e2e/goal/createGoals.cy.ts

```diff
@@ -17,6 +17,7 @@ context('Create goals', () => {
     cy.task('getActionPlan')
     cy.task('stubLearnerProfile')
     cy.task('stubLearnerEducation')
+    cy.task('stubGetAllPrisons')
```

Comment:
Not strictly related to this PR, but this was a missing stub from this cypress test suite (resulting in some 404s but not breaking the journey as failing to lookup the prison name is handled and is acceptable)